Makefile

```diff
@@ -8,7 +8,7 @@ INSTALL_DIR      ?= /usr/local
 vendor:
```

Comment:
> make build && sudo make install or something like make build && INSTALL_DIR=~ make install

This is what I'd expect coming to a Makefile based project, I don't know if the go community does things differently. `make install` normally installs to a a system location to be overridden by PREFIX for c/c++ stuff.